docs/index.html

```diff
@@ -186,7 +186,14 @@ <h5 class="d-none d-md-block">&nbsp;</h5>
             </div>
             
 
-          
+            <div class="form-check">
+              <input class="form-check-input" type="radio" name="server" id="server-kubelet" value="kubelet" >
+              <label class="form-check-label" for="server-kubelet">
+                Kubelet/Kubernetes
+              </label>
+            </div>
+
+
```

Comment:
For changes to the `index.html` you'll want to make them in `templates/index.ejs` as `index.html` is rendered from the template.

Reply:
@atombrella This will be generated from configs.js values